Modules/SwiftUISupport/SwiftUIContainerView.swift

```diff
@@ -0,0 +1,62 @@
+import JamitFoundation
+import SwiftUI
+import UIKit
+
+@available(iOS 13.0, *)
+/// Takes a swiftui view as a generic type and wraps it into a `StatefulView`.
+public class SwiftUIContainerView<ContentView: View>: StatefulView<EmptyViewModel> {
```

Comment:
I'm probably missing something, but at the moment I don't see how this type offers any benefits over directly using `UIHostingController`. Is it really needed?